### PR TITLE
Moved calendar.yaml to calendar/calendar.yaml

### PR DESCRIPTION
**Pull request by karlschriek**
Added an OWNERS file for calendar

### CODE REVIEW

**Comment by karlschriek** [comment]
@googlebot I fixed it.

File: calendar/OWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+approvers:
+  - abhi-g
```
Comment (jlewi) [inline calendar/OWNERS:2]: Only list the people that have explicitly ack'd being calendar admins. Based on #342 that would just be you.

Comment (karlschriek) [reply calendar/OWNERS:2]: Also jfogarty and yuzliu (see https://github.com/kubeflow/community/issues/342)

The others were in the root OWNERS file, should I remove them fro here?

Comment (karlschriek) [reply calendar/OWNERS:2]: Now also my colleague nicholas-abad

Comment (jlewi) [reply calendar/OWNERS:2]: Yes. Only list the folks that are specifically helping out with the calendar.
Lets cap it at the 4 people you mentioned.

Please alphabetize

**Comment by jlewi** [comment]
@karlschriek ping?

File: calendar/OWNERS (hunk 1 above)
Comment (jlewi) [reply calendar/OWNERS:2]: @karlschriek please remove anyone who isn't already a member of the Kubeflow GitHub org.

**Comment by karlschriek** [comment]
@jlewi, done, also https://github.com/kubeflow/internal-acls/pull/277

**Comment by karlschriek** [comment]
> The following users are mentioned in OWNERS file(s) but are untrusted for the following reasons. One way to make the user trusted is to add them as [members](https://github.com/kubeflow/kubeflow/blob/master/CONTRIBUTING.md) of the kubeflow org. You can then trigger verification by writing `/verify-owners` in a comment.
> 
> * karlschriek
>   
>   * User is not a member of the org. User is not a collaborator. Satisfy at least one of these conditions to make the user trusted.
> * nicholas-abad
>   
>   * User is not a member of the org. User is not a collaborator. Satisfy at least one of these conditions to make the user trusted.
> * jfogarty
>   
>   * User is not a member of the org. User is not a collaborator. Satisfy at least one of these conditions to make the user trusted.

@jlewi I think this is preventing the merge

**Comment by jlewi** [comment]
@karlschriek You should remove anyone who is not a member of the GitHub org from the OWNERs file.

**Comment by jlewi** [comment]
@karlschriek It looks like kubeflow/community#365 will got merged first so you will need to rebase of the latest master and make sure you pull in those changes.

**Comment by karlschriek** [comment]
> @karlschriek You should remove anyone who is not a member of the GitHub org from the OWNERs file.

That would mean removing three of the four people (including myself) who have volunteered to act as calendar admins. Isn't the solution to rather add those people as members of the Kubeflow org?

**Comment by jlewi** [comment]
@karlschriek Yes people should become members of kubeflow.org before they become calendar admins.

@jtfogarty  Is already a member of the org.  It looks like you have a typo.

Given your participation so far, if you submit a request to join the org I'll approve it.

The others should become members of the GitHub org before applying to become calendar admins. They can still help out e.g. by submitting PRs to update the code.

**Comment by jlewi** [comment]
Looks like its out of sync.
@karlschriek The way to reduce toil and get others to do the heavy lifting is to automate the sync
https://github.com/kubeflow/community/issues/274

**Comment by jlewi** [comment]
@karlschriek ping?

**Comment by jlewi** [comment]
@karlschriek any update ton this?

**Comment by karlschriek** [comment]
Yeah, sorry. Little bit under water at the moment. Where can I submit a request to join the github org? Once that is done I can proceed with this.

**Comment by jlewi** [comment]
@karlschriek 
https://www.kubeflow.org/docs/about/contributing/

**Comment by jlewi** [comment]
@karlschriek Looks like you still have folks who aren't github org members

**Comment by karlschriek** [comment]
@jlewi seems ready to go now

**Pull request merged**